qbittorrentapi/torrents.pyi

```diff
@@ -404,6 +406,17 @@ class Torrents(ClientCache):
             tag: Optional[Text] = None,
             **kwargs: KwargsT
         ) -> TorrentInfoList: ...
+        def seeding(
+            self,
+            category: Optional[Text] = None,
+            sort: Optional[TorrentStatusesT] = None,
```

Comment:
I just noticed that `sort` is using `TorrentStatusesT` when it definitely should not be...

Do you mind updating `sort` in each of these?

```suggestion
            sort: Optional[Text] = None,
```